polly/include/polly/CodeGen/IRBuilder.h

```diff
@@ -60,7 +60,8 @@ class ScopAnnotator {
 
   /// Annotate the loop latch @p B wrt. @p L.
   void annotateLoopLatch(llvm::BranchInst *B, llvm::Loop *L, bool IsParallel,
-                         bool IsLoopVectorizerDisabled) const;
+                         bool setVectorizeMetadata,
```

Comment:
```suggestion
                         bool SetVectorizeMetadata,
```
[LLVM uses CamelCase for variable names](https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly)